OpenRA.Mods.Common/Activities/Hunt.cs

```diff
@@ -36,7 +36,7 @@ public override bool Tick(Actor self)
 			if (IsCanceling)
 				return true;
 
-			var targetActor = targets.ClosestTo(self);
+			var targetActor = targets.ClosestToWithPathFrom(self);
```

Comment:
hunt will also need distance checks so that  targets on different terrain don't become invalid

Reply:
Since it's not caching the target I believe it is always choosing a fresh, valid target. I'm not sure what you'd want me to change here.